administrator/components/com_content/controllers/article.php

```diff
@@ -155,30 +155,47 @@ public function batch($model = null)
 	 *
 	 * @since	3.1
 	 */
-	protected function postSaveHook(JModelLegacy $model, $validData = array())
+	protected function postSaveHook(JModelLegacy $model, $validData)
 	{
 		JModelLegacy::addIncludePath(JPATH_ADMINISTRATOR . '/components/com_menus/models', 'MenusModel');
-		$itemmodel = JModelAdmin::getInstance('Item', 'MenusModel', array());
+		$itemModel = JModelAdmin::getInstance('Item', 'MenusModel', array());
+		$itemModel->addTablePath(JPATH_ADMINISTRATOR . '/components/com_menus/tables');
+
+		$db = JFactory::getDbo();
+		$query = $db->getQuery(true);
+		$query->select('extension_id');
+		$query->from($db->quoteName('#__extensions'));
+		$query->where($db->quoteName('name') . " = " . $db->quote('com_content'));
```

Comment:
@shubhamnba2009 No, there isn't imo. JTableUcm is something completely different (related to #__ucm (Unified content model) tables). But you can write yourself a helper function for that. DRY - Don't repeat your self. :-)

P.S.: You can chain a $query object. 
```php
$query->select('id')
   ->from('#__content')
   ->where($db->qn('title') . ' = ' . $db->q(..).
```

$db->q and $db->qn are nice shortcuts to keep your code more readable.